rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc

```diff
@@ -0,0 +1,40 @@
+= RPA Builder 6.5.6 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== February 6, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The Tomcat dependency is upgraded to version 9.0.85 to address reported security vulnerabilities.
```

Comment:
Tomcat upgrade was already contained in RPA Builder Version 6.5.4

Reply:
You are correct, this was a duplicate, just removed it. Thanks!